scalableminds/webknossos
# rename sbt project and binary from “oxalis” to “webknossos”

let’s give it a shot…

### URL of deployed dev instance (used for testing):
- https://pruneoxalis.webknossos.xyz

### Steps to test:
- CI should run through
- should be deployable and start correctly

------
